packages/webapp/src/components/Form/Radio/index.jsx

```diff
@@ -18,6 +19,9 @@ const Radio = ({
   ...props
 }) => {
   const name = hookFormRegister?.name ?? props?.name;
+  const pill = props?.pill;
+  const checked = props?.checked;
```

Comment:
same I think we could add this to explicit props?

Reply:
Now it's destructured too!